src/main/kotlin/no/nav/familie/ba/migrering/services/HentSakTilMigreringService.kt

```diff
@@ -29,29 +29,33 @@ class HentSakTilMigreringService(
         val personerForMigrering = infotrygdClient.hentPersonerKlareForMigrering(
             MigreringRequest(
                 page = 1,
-                size = MAX_PERSON_FOR_MIGRERING,
+                size = ANTALL_PERSONER_SOM_HENTES_FRA_INFOTRYGD,
                 valg = "OR",
                 undervalg = "OS",
                 maksAntallBarn = 1,
             )
         )
 
-        if (personerForMigrering.size > MAX_PERSON_FOR_MIGRERING) {
+        if (personerForMigrering.size > ANTALL_PERSONER_SOM_HENTES_FRA_INFOTRYGD) {
             Log.error("For manger personer (${personerForMigrering.size}) avbryter migrering")
             return
         }
 
         Log.info("Fant ${personerForMigrering.size} personer for migrering")
 
-        personerForMigrering.forEach {
+        var antallPersonerMigrert = 0
+        personerForMigrering.takeWhile {antallPersonerMigrert < MAX_ANTALL_PERSONER_SOM_SKAL_MIGRERES}.forEach {
             if (!migrertsakRepository.existsByPersonIdentAndStatus(it, MigreringStatus.MIGRERT_I_BA)) {
                 taskRepository.save(MigreringTask.opprettTask(MigreringTaskDto(it)))
+                antallPersonerMigrert++
             }
+
         }
```

Comment:
Jeg så at det ble opprettet 53 nye MigreringTask'er i dag, så da fungerer det nok ikke med `takeWhile`
```suggestion
        for (person in personerForMigrering) {
            if (!migrertsakRepository.existsByPersonIdentAndStatus(person, MigreringStatus.MIGRERT_I_BA)) {
                taskRepository.save(MigreringTask.opprettTask(MigreringTaskDto(person)))
                antallPersonerMigrert++
            }
            if (antallPersonerMigrert == MAX_ANTALL_PERSONER_SOM_SKAL_MIGRERES)
                break
        }
```